### PR TITLE
removing _burn and _mint function

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: contracts/Token.sol [hunk 1]
```diff
@@ -1,39 +0,0 @@
-// SPDX-License-Identifier: MIT
-pragma solidity >=0.6.0 <0.8.0;
```
Comment (AndreiNavoichyk) [inline contracts/Token.sol]: Please, revert this SC back. This is Cere ERC20 SC. We can not remove/update it.

Comment (karthikeyan-cere) [reply contracts/Token.sol]: okay.

File: contracts/handlers/ERC721Handler.sol [hunk 2]
```diff
@@ -118,11 +112,7 @@ contract ERC721Handler is IDepositExecute, HandlerHelpers, ERC721Safe {
             metaData = bytes(erc721.tokenURI(tokenID));
         }
 
-        if (_burnList[tokenAddress]) {
-            burnERC721(tokenAddress, tokenID);
-        } else {
-            lockERC721(tokenAddress, depositer, address(this), tokenID);
-        }
+        lockERC721(tokenAddress, depositer, address(this), tokenID);
```
Comment (AndreiNavoichyk) [inline contracts/handlers/ERC721Handler.sol:114]: Update the method description in line 90.

File: contracts/handlers/ERC20Handler.sol [hunk 3]
```diff
@@ -101,11 +95,7 @@ contract ERC20Handler is IDepositExecute, HandlerHelpers, ERC20Safe {
         address tokenAddress = _resourceIDToTokenContractAddress[resourceID];
         require(_contractWhitelist[tokenAddress], "provided tokenAddress is not whitelisted");
 
-        if (_burnList[tokenAddress]) {
-            burnERC20(tokenAddress, depositer, amount);
-        } else {
-            lockERC20(tokenAddress, depositer, address(this), amount);
-        }
+        lockERC20(tokenAddress, depositer, address(this), amount);
```
Comment (AndreiNavoichyk) [inline contracts/handlers/ERC20Handler.sol:97]: Update the method description in line 79.

**Review by AndreiNavoichyk — APPROVED** [review]

File: contracts/handlers/ERC721Handler.sol (hunk 2 above)
Comment (karthikeyan-cere) [reply contracts/handlers/ERC721Handler.sol:114]: Done

File: contracts/handlers/ERC20Handler.sol (hunk 3 above)
Comment (karthikeyan-cere) [reply contracts/handlers/ERC20Handler.sol:97]: Done

**Pull request merged**